providers/porkbun/api.go

```diff
@@ -121,3 +127,26 @@ func (c *porkbunProvider) getRecords(domain string) ([]domainRecord, error) {
 	}
 	return records, nil
 }
+
+func (c *porkbunProvider) getNameservers(domain string) ([]string, error) {
+	params := requestParams{}
+	var bodyString, err = c.post(fmt.Sprintf("/domain/getNs/%s", domain), params)
+	if err != nil {
+		return nil, fmt.Errorf("failed fetching nameserver list from porkbun: %s", err)
```

Comment:
Please use %w for errors in fmt.Errorf.

```suggestion
		return nil, fmt.Errorf("failed fetching nameserver list from porkbun: %w", err)
```

Reply:
Happy to do so! I copied this line from elsewhere in the provider, and the prevailing style across the entire codebase appears to be `%s`, but I'll only alter the instances in this provider file.